systemtest/loadgen/eventhandler/handler.go

```diff
@@ -155,6 +155,9 @@ type Config struct {
 	// RewriteTransactionTypes controls the rewriting of `transaction.type`
 	// in events.
 	RewriteTransactionTypes bool
+
+	// ReplaceGlobalLabels replaces metadata.labels with the specified string.
+	ReplaceGlobalLabels string
```

Comment:
Can we use `RewriteGlobalLabels`? I know that it will be slightly different than other flags in the way that it will rewrite with the provided input but it still performs a similar functionality.

Reply:
Renamed